### PR TITLE
SSO: Better alignment of "W" icon

### PR DESCRIPTION
**Pull request by ebinnion**
This PR fixes the align of the WordPress icon so that it is center with the "Login with WordPress.com" text. Also, this PR adds `modules/sso/jetpack-sso-login.css' to the build process so that we create an RTL stylesheet for the SSO CSS that is loaded on `wp-login.php`

cc @lezama and @dereksmart to test. Pinging @dereksmart since I am changing the `gulpfile.js`, and when blaming, it looked like he was the last to touch a section I changed.

cc @michaelarestad for design review.

To test:

- Checkout `update/jetpack-sso-styles` branch
- Run `gulp` to build RTL stylehseets
- Ensure that site is connected to WP.com and you have SSO enabled
- Go to `$site/wp-admin`
- You should see something a bit like the first screenshot below (without the logged out notice, of course)
- Once logged in, go to settings and change your site's language to an RTL language (Hebrew, for example
- Log out
- You should now see something like the second screenshot below
  
#### Not RTL
screen shot 2016-06-08 at 5 14 08 pm
#### RTL
screen shot 2016-06-08 at 5 14 31 pm

#### Not RTL
screen shot 2016-06-08 at 5 15 50 pm

#### RTL
screen shot 2016-06-08 at 5 14 58 pm

### CODE REVIEW

**Comment by MichaelArestad** [comment]
Looking good!
I would move the logo up 1px (-3px). That's it!

One unrelated note:
* It would be good to make the lighter weight of the font in "Log in as" `font-weight: 300`. It's a tad too light on retina screens as is.

File: modules/sso.php [hunk 1]
```diff
@@ -181,7 +181,12 @@ public function login_enqueue_scripts() {
 			return;
 		}
 
-		wp_enqueue_style( 'jetpack-sso-login', plugins_url( 'modules/sso/jetpack-sso-login.css', JETPACK__PLUGIN_FILE ), array( 'login', 'genericons' ), JETPACK__VERSION );
+		if( is_rtl() ){
```
Comment (lezama) [inline modules/sso.php]: spaces missing, after `if` and before `{` :P

**Comment by lezama** [comment]
Code wise looks 👍

**Comment by dereksmart** [comment]
Looks fine to me 👍

**Comment by ebinnion** [comment]
Here are some updated screenshot after making changes:

screen shot 2016-06-09 at 10 22 58 am
screen shot 2016-06-09 at 10 23 30 am

File: modules/sso/jetpack-sso-login.css [hunk 2]
```diff
@@ -119,33 +119,29 @@
...
-	content: '\f205';
-	color: #fff;
-	-webkit-font-smoothing: antialiased;
-	width: 48px;
-	height: 107%;
-	height: calc( 100% + 2px );
-	font: normal 24px/1 Genericons !important;
-	text-shadow: none;
+	.jetpack-sso.button .genericon-wordpress {
+		top: -2px;
```
Comment (MichaelArestad) [inline modules/sso/jetpack-sso-login.css]: Can you make this `-3px`? After that, :shipit:

**Pull request merged**